src/components/App/App.test.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @file App.test.js
+ * Contains tests for App.js.
+ */
+
+import React from 'react';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+
+import App from './App';
+
+describe('<App />', () => {
+  it('Matches its snapshot', () => {
+    const store = configureStore()({
```

Comment:
Note my approach to mocking a redux store.

Reply:
I'm cool with this.